server/bundler/hot-reloader.js

```diff
@@ -8,6 +8,12 @@
  */
 const socketio = require( 'socket.io' );
 const debug = require( 'debug' )( 'calypso:bundler:hot-reloader' );
+const { memoize } = require( 'lodash' );
+memoize.Cache = WeakMap; // play better with the GC over time
```

Comment:
This will replace the cache constructor for all `memoize` instances across the Calypso server. This should also work and is more localized:
```js
const getStats = memoize( stats => stats.toJson() );
getStats.cache = new WeakMap();
```

Reply:
> This will replace the cache constructor for all memoize instances across the Calypso server.

Is that really a bad thing? 😈 

Thanks, I'll make the update.

Reply:
played around with this a bit and it's actually a bit scary.

```
const f = memoize( num => expensiveThing( num ) )
f.cache = new WeakMap()

f(5)
// Uncaught TypeError: Invalid value used as weak map key
```

So WeakMap isn't a great general purpose replacement. 😱

Reply:
It's fine in this case though because stats is always an object, which is a fine key into a weakmap.